acceptance/terraform/test.toml

```diff
@@ -1,3 +1,7 @@
+[[Repls]]
```

Comment:
Needed because with OS replacement in ubuntu `linux_amd64` is converted to `[OS}_amd64`